src/server/util/API.js

```diff
@@ -103,12 +134,17 @@ async function getIssuesNumber(organization, contributor) {
 async function getContributorInfo(organization, contributor) {
     const home = BASEURL + '/' + contributor
     const avatarUrl = await getContributorAvatar(contributor)
-    const openPRsNumber = await getOpenPRsNumber(organization, contributor)
-    const openPRsLink = `${BASEURL}/search?q=type:pr+org:${organization}+author:${contributor}+is:open+created:>=${Config.startDate}`
-    const mergedPRsNumber = await getMergedPRsNumber(organization, contributor)
-    const mergedPRsLink = `${BASEURL}/search?q=type:pr+org:${organization}+author:${contributor}+is:merged+created:>=${Config.startDate}`
-    const issuesNumber = await getIssuesNumber(organization, contributor)
-    const issuesLink = `${BASEURL}/search?q=type:issue+org:${organization}+author:${contributor}+created:>=${Config.startDate}`
+    const openPRsNumber = await getOpenPRsNumber(organization, contributor, Config.includedRepositories)
```

Comment:
You can make a string of appended repos at once here by traversing the included repos array and pass it on to all the 3 getNumber functions. This will help remove the repeated code and save time.